server/src/main/java/io/crate/types/DataTypes.java

```diff
@@ -253,6 +253,8 @@ public static void toStream(DataType type, StreamOutput out) throws IOException
             return valueFromList((List) value);
         } else if (value.getClass().isArray()) {
             return valueFromList(Arrays.asList((Object[]) value));
+        } else if (value instanceof StringType) {
+            return STRING;
```

Comment:
Would it make sense to add a `String.class` entry to `POJO_TYPE_MAPPING` instead?